Sources/PeckerKit/Configuration.swift

```diff
@@ -77,6 +81,8 @@ public struct Configuration {
         let excluded = (yamlConfiguration?.excluded ?? []).map {
             return AbsolutePath($0, relativeTo: projectPath)
         }.filter { localFileSystem.exists($0) }
+
+        let xcodeAppPath = createXcodeAppPath(path: yamlConfiguration?.xcodeAppPath, fileSystem: localFileSystem) ?? AbsolutePath("/Applications/Xcode.app")
```

Comment:
`AbsolutePath("/Applications/Xcode.app")`:  what do you think of making this an even smarter default by running `xcode-select -p` and getting the path :)

Reply:
Good point. `PeckerKit` is probably not the right place to add a call to `xcode-select`. Maybe we can add an option to the commandline tool and then you can call the tool with `pecker --xcode-app-path $(xcode-select -p)`. What do you think?

Reply:
The output of `xcode-select -p` includes `.../Contents/Developer/`. Should we be smart and check the given path and append `/Contents/Developer/` if it is missing? 

While writing I already don't like the idea. 😄 I think requiring the path that `xcode-select -p` gives you is the better idea.